package.json

```diff
@@ -2,41 +2,51 @@
   "name": "@chainlink/token",
   "homepage": "https://github.com/smartcontractkit/LinkToken",
   "repository": "https://github.com/smartcontractkit/LinkToken",
-  "version": "1.1.0",
+  "version": "1.2.0-alpha",
   "license": "MIT",
   "authors": [
-    "Steve Ellis <steve@smartcontract.com>",
-    "Kristijan Rebernisak <kristijan@smartcontract.com>"
+    "Steve Ellis <steve@smartcontract.com>"
   ],
-  "devDependencies": {
-    "@chainlink/belt": "^0.0.3",
-    "@chainlink/contracts": "^0.1.6",
-    "@truffle/hdwallet-provider": "^1.0.43",
-    "bignumber.js": "^2.0.0",
-    "eth-gas-reporter": "^0.2.17",
-    "ganache-cli": "^6.4.1",
-    "moment": "^2.15.2",
-    "solc": "^0.6.0",
-    "truffle": "^5.1.32",
-    "web3": "^1.3.0"
-  },
   "files": [
-    "README.md",
-    "contracts"
+    "build/**/*.js",
+    "build/contracts/*",
+    "build/artifacts/**/*.json",
+    "build/types/**/*.ts",
+    "contracts/*"
   ],
   "scripts": {
-    "compile": "belt compile all",
-    "clean": "rm -r build/contracts build/ethers build/truffle",
-    "compile:clean": "yarn clean & yarn compile",
-    "test": "env NODE_ENV=test truffle test",
-    "test:v0.4": "env NODE_ENV=test find ./test/v0.4 -type f | xargs truffle test",
-    "test:v0.6": "env NODE_ENV=test find ./test/v0.6 -type f | xargs truffle test",
-    "migrate:v0.4": "env VERSION=v0.4 truffle migrate",
-    "migrate:v0.6": "env VERSION=v0.6 truffle migrate"
+    "clean": "tsc -b --clean tsconfig.json && rm -rf build/",
+    "setup": "yarn build:contracts",
+    "build:contracts": "yarn build:contracts:v0.4 && yarn build:contracts:v0.6",
+    "build:contracts:v0.4": "VERSION=v0.4 hardhat compile --show-stack-traces",
+    "build:contracts:v0.6": "VERSION=v0.6 hardhat compile --show-stack-traces",
+    "test": "hardhat test --show-stack-traces"
```

Comment:
We can add a simple Github CI workflow that runs yarn install, setup and test now :)

Reply:
Yes, but in another PR :) The work on this started [here](https://github.com/smartcontractkit/LinkToken/pull/29/commits/346e1598a4f5337b56fc71b81b8c0be0c2c54fcd).